typescript/sdk/src/gas/token-prices.test.ts

```diff
@@ -1,14 +1,15 @@
 import { expect } from 'chai';
 import sinon from 'sinon';
 
-import { ethereum, solanamainnet } from '@hyperlane-xyz/registry';
-
 import { TestChainName, testChainMetadata } from '../consts/testChains.js';
 
 import { CoinGeckoTokenPriceGetter } from './token-prices.js';
 
 const MOCK_FETCH_CALLS = true;
 
+const ethereum: { name?: string } = {};
+const solanamainnet: { name?: string } = {};
```

Comment:
surprised that the only thing needed from here is the name

Reply:
Actually, I'll remove that. It was needed because a few lines of code in this file used `ethereum.name`, etc. But I changed that to constants and the remaining test code doesn't exercise `CoinGeckoTokenPriceGetter` enough to read any of the fields.

Once we get to improving these tests, we can add fuller chain configs to `testChains.ts` and use them.

Reply:
Actually, can't remove these because `CoinGeckoTokenPriceGetter` expects the chain objects to be present in the chainmap, even if the objects themselves are empty.